src/ResponseSender/SimpleStreamResponseSender.php

```diff
@@ -23,9 +24,16 @@ public function sendStream(SendResponseEvent $event)
             return $this;
         }
         $response = $event->getResponse();
+
+        if (! $response instanceof Stream) {
+            throw new RuntimeException('Event is not an instance of ' . Stream::class);
```

Comment:
Please add the type of `$response` here

Also: `Stream`?

Reply:
Yes, it's the only response type with `getStream()` method.

There is a chain to send the response, the first one is `PhpEnvironmentResponseSender` which has a method to check if the event response is an instance of `Zend\Http\PhpEnvironment\Response`.

I added the check also for the `SimpleStreamResponseSender`.